app.js

```diff
@@ -1,17 +1,105 @@
 /* Imports */
-// import { getRandomItem } from './utils.js';
+//import { getRandomItem } from './utils.js';
 
 /* State */
+let gameState = 'guess'; //'guess' or 'results'
+let guess = ''; //'guess-1', 'guess-2', or 'guess-3'
+let pearl = '';
 
 /* Actions */
-function loadPage() {}
+function loadPage() {
+    displayShells();
+}
+
+function playAgain() {
```

Comment:
This is setting the state of the game but isn't resetting the rest of the state to allow us to guess again. Can you think of how we would do that? Let me know if you have questions about this.